modules/test/playwright/tests/frontend-data-set-views-web/pages/ViewsPage.ts

```diff
@@ -51,5 +51,9 @@ export class ViewsPage {
 			})
 			.first()
 			.click();
+
+		await this.page
+			.getByRole('heading', {name: 'Data Set View Sample'})
+			.waitFor();
```

Comment:
This is what I meant in some previous comment! We might need to apply this in more places

Reply:
Does this work? :thinking: [Docs](https://playwright.dev/docs/api/class-locator#locator-wait-for) say that timeout `Defaults to 0 - no timeout`, and we don't have `actionTimeout` in config. Doesn't that mean that `.waitFor()` does nothing?